<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STACalculatorDialog</class>
 <widget class="QDialog" name="STACalculatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>585</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>585</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STA Calculator</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>8</y>
     <width>549</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Time">
    <attribute name="title">
     <string>Time</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>8</y>
       <width>529</width>
       <height>213</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gregorian date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2014</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Julian date UTC:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="JDUTCEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Modified Julian date UTC:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="MJDUTCEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Julian date TDB:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="JDTDBEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Modified Julian date TDB:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="MJDTDBEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Day fraction [s]:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="DayFractionEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Planets">
    <attribute name="title">
     <string>Planets</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>16</y>
       <width>521</width>
       <height>197</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Central body: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxPlanets">
        <item>
         <property name="text">
          <string>Earth</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboEarth.png</normaloff>:/icons/ComboEarth.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Moon</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboMoon.png</normaloff>:/icons/ComboMoon.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mercury</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboMercury.png</normaloff>:/icons/ComboMercury.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mars</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboMars.png</normaloff>:/icons/ComboMars.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jupiter</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboJupiter.png</normaloff>:/icons/ComboJupiter.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saturn</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboSaturn.png</normaloff>:/icons/ComboSaturn.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Uranus</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboUranus.png</normaloff>:/icons/ComboUranus.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Neptune</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboNeptune.png</normaloff>:/icons/ComboNeptune.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pluto</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboPluto.png</normaloff>:/icons/ComboPluto.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sun</string>
         </property>
         <property name="icon">
          <iconset resource="../../iconary/sta-icons.qrc">
           <normaloff>:/icons/ComboSun.png</normaloff>:/icons/ComboSun.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Enceladus</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/ComboEnceladus.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Europa</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/ComboEuropa.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Titan</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/ComboTitan.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Triton</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/ComboTriton.png</normalon>
          </iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Average radius [Km]:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Period [days]:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Inclination [deg]:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Average distance to Sun [Km]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="AverageRadiusEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="PeriodEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="InclinationEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="DistanceSunEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="gravityConstantEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Gravity constant [Km3/s2]:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>272</y>
     <width>93</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>JDUTCEdit</tabstop>
  <tabstop>MJDUTCEdit</tabstop>
  <tabstop>JDTDBEdit</tabstop>
  <tabstop>MJDTDBEdit</tabstop>
  <tabstop>DayFractionEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>comboBoxPlanets</tabstop>
  <tabstop>AverageRadiusEdit</tabstop>
  <tabstop>PeriodEdit</tabstop>
  <tabstop>InclinationEdit</tabstop>
  <tabstop>DistanceSunEdit</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../iconary/sta-icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>JDUTCEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>MJDUTCEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>JDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>MJDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>DayFractionEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MJDUTCEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>JDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MJDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DayFractionEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>JDUTCEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MJDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>JDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDUTCEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DayFractionEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>JDUTCEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MJDUTCEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MJDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DayFractionEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>JDUTCEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>JDTDBEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MJDTDBEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DayFractionEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxPlanets</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>AverageRadiusEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxPlanets</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>PeriodEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxPlanets</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InclinationEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxPlanets</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>DistanceSunEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>STACalculatorDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
